stories/fixtures/long-file-sample.tsx

```diff
@@ -0,0 +1,193 @@
+export default `/* @flow */
```

Comment:
It's a bit funny (as in "ha ha" funny, not "odd" funny) that our sample uses `flow`. Not a change request, just an observation.